docs/developer-docs/integrations/sns/integrate-sns/ledger-integration.md

```diff
@@ -1 +1,69 @@
-Title h1: SNS Ledger Integration
\ No newline at end of file
+# Ledger canister
+
+The ledger canister keeps track of the transactions between accounts.
+
+## Initialisation
+
+The index canister initialisation requires the following 
+parameters to be initialized:
+
+```
+type InitArgs = record {
+    minting_account : Account;
+    transfer_fee : nat64;
+    token_symbol : text;
+    token_name : text;
+    metadata : vec record { text; Value };
+    initial_balances : vec record { Account; nat64 };
+    archive_options : record {
+        num_blocks_to_archive : nat64;
+        trigger_threshold : nat64;
+        max_message_size_bytes : opt nat64;
+        cycles_for_archive_creation : opt nat64;
+        node_max_memory_size_bytes : opt nat64;
+        controller_id : principal;
+    };
+};
+```
+
+Example with `dfx`:
+
+```shell
+dfx deploy icrc1-ledger --argument "(record {
+    token_symbol = \"ABC\";
+    token_name = \"My example token\";
+    minting_account = record { owner = principal \"rrkah-fqaaa-aaaaa-aaaaq-cai\" };
+    transfer_fee = 10_000;
+    metadata = vec {};
+    initial_balances = vec {};
+    archive_options = record {
+        num_blocks_to_archive = 10_000;
+        trigger_threshold = 20_000;
+        controller_id = principal \"rrkah-fqaaa-aaaaa-aaaaq-cai\";
+        cycles_for_archive_creation = opt 4_000_000_000_000;
+    };
+})"
+```
+
+## Usage
+
+Here are the supported methods from the ledger canister:
```

Comment:
should we also add a sentence to each of them what it does? 
If this is described elsewhere, I think we can also link there

Reply:
Added a quick description for each method.